### ISSUE TITLE
running from source, sqlite3 "no column named content"

### ISSUE THREAD

**Issue by xu4wang** [foobuzz/todo#14]
Just cloned todo, installed python3, test.py got no error, but when I tried to add one todo item, it complains  "no column named content".

Do you know how should I fix this? Thanks.

```

WangXus-MacBook-Pro:source wangxu$ ./test.py
* Unit tests
..............
----------------------------------------------------------------------
Ran 14 tests in 0.010s

OK
WangXus-MacBook-Pro:source wangxu$ ./todo.py
WangXus-MacBook-Pro:source wangxu$ ./todo.py add "fix the stuff"
Traceback (most recent call last):
  File "./todo.py", line 5, in <module>
    main()
  File "/Users/wangxu/Documents/github/todo/source/todo/todo.py", line 128, in main
    result = dispatch(args, daccess)
  File "/Users/wangxu/Documents/github/todo/source/todo/todo.py", line 396, in dispatch
    return DISPATCHER[args['command']](args, daccess)
  File "/Users/wangxu/Documents/github/todo/source/todo/todo.py", line 173, in add_task
    id_ = daccess.add_task(title, content, context, options)
  File "/Users/wangxu/Documents/github/todo/source/todo/data_access.py", line 226, in add_task
    c.execute(query, values)
sqlite3.OperationalError: table Task has no column named content
WangXus-MacBook-Pro:source wangxu$ ls -al .toduh
total 64
drwxr-xr-x   4 wangxu  staff    128 Jul  7 12:23 .
drwxr-xr-x  10 wangxu  staff    320 Jul  6 21:40 ..
-rw-r--r--   1 wangxu  staff  28672 Jul  7 12:18 data.sqlite
-rw-r--r--   1 wangxu  staff      3 Jul  7 12:18 version
WangXus-MacBook-Pro:source wangxu$


```

**Comment by xu4wang** [comment]
Finally I tried 

```
pip3 install todocli
```

However it was not fixed.

```

WangXus-MacBook-Pro:~ wangxu$ todo
WangXus-MacBook-Pro:~ wangxu$ todo add "fix the stuff"
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/bin/todo", line 10, in <module>
    sys.exit(main())
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/todo/todo.py", line 128, in main
    result = dispatch(args, daccess)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/todo/todo.py", line 396, in dispatch
    return DISPATCHER[args['command']](args, daccess)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/todo/todo.py", line 173, in add_task
    id_ = daccess.add_task(title, content, context, options)
  File "/usr/local/Cellar/python/3.7.3/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/todo/data_access.py", line 226, in add_task
    c.execute(query, values)
sqlite3.OperationalError: table Task has no column named content
WangXus-MacBook-Pro:~ wangxu$

```

**Comment by foobuzz** [comment]
Hello,

Thanks for reporting this issue. Could you post the output of the following commands please:

 - `todo --version`
 - `cat .toduh/version`
 - `sqlite3 .toduh/data.sqlite '.schema task'`

Thanks!

**Comment by xu4wang** [comment]
Thank for quick response and for the cli todo app. I like it. 
I figured out the root cause is in init_db.py, there is one SQL statement

```
	INSERT INTO `Context` (path) VALUES ('')

```
Which fails silently during update_database. I fixed it by set isolation_level=None in my clone and made one pull request.

Anyway please find below the output of my PIP3 installation.

```
WangXus-MacBook-Pro:source wangxu$ todo --version
3.2
WangXus-MacBook-Pro:source wangxu$ cd
WangXus-MacBook-Pro:~ wangxu$ cat .toduh/version
3.2WangXus-MacBook-Pro:~ wangxu$ sqlite3 .toduh/data.sqlite '.schema task'
CREATE TABLE `Task` (
		`id`	INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT UNIQUE,
		`title`	TEXT NOT NULL,
		`created`	TEXT NOT NULL DEFAULT (datetime('now')),
		`deadline`	TEXT,
		`start`	TEXT NOT NULL DEFAULT (datetime('now')),
		`priority`	INTEGER NOT NULL DEFAULT 1,
		`done`	TEXT,
		`context`	INTEGER NOT NULL REFERENCES Context(id) ON DELETE CASCADE
	);
WangXus-MacBook-Pro:~ wangxu$

```

**Comment by MatthewFlamm** [comment]
I have this issue too.

**Comment by MatthewFlamm** [comment]
Also tried fresh install and just typing "todo add test" using py3.7.3 and the latest todocli on pypi.  Same error on raspbian

**Comment by foobuzz** [comment]
@xu4wang Thank you for your contribution! The fix is available in version 3.2.1.

**Comment by xu4wang** [comment]
> @xu4wang Thank you for your contribution! The fix is available in version 3.2.1.

thank you!  :-)
